src/app/designationEditor/designationEditor.spec.js

```diff
@@ -176,6 +176,24 @@ describe('Designation Editor', function () {
       $httpBackend.flush()
     })
 
+    it('to sign in and try again if the response is 422', () => {
+      $ctrl.designationNumber = designationSecurityResponse.designationNumber
+
+      jest.spyOn($ctrl.sessionModalService, 'open').mockReturnValue(() => ({ result: $q.resolve() }))
+      $httpBackend.expectGET(designationConstants.designationEndpoint + '?designationNumber=' + designationSecurityResponse.designationNumber)
+        .respond(422, null)
+      $httpBackend.expectGET(designationConstants.designationImagesEndpoint + '?designationNumber=' + designationSecurityResponse.designationNumber)
+        .respond(422, null)
+
+      $ctrl.getDesignationContent().then(() => {
+        expect($ctrl.sessionModalService.open).toHaveBeenCalled()
+        expect($ctrl.designationContent).toBeDefined()
+        expect($ctrl.contentLoaded).toEqual(true)
+        expect($ctrl.loadingContentError).toEqual(false)
```

Comment:
I wonder if these expectations are being called or if we need to add `done()` to this test since it is asynchronous. Did you test to find out for sure if these are being called?